### PR TITLE
E2E-602: offender address type

### PR DESCRIPTION
**Pull request by axle-h**
New fields returned on offender address as in Delius:
* type: latest personal circumstances sub type
* type verified: latest personal circumstances evidenced flag
* latest assessment date: latest address assessment, assessment date
* created & updated datetimes

### CODE REVIEW

File: src/main/java/uk/gov/justice/digital/delius/data/api/Address.java [hunk 1]
```diff
@@ -7,24 +7,64 @@
...
+    @ApiModelProperty(name = "Address type")
+    private KeyValue type;
+
+    @ApiModelProperty(name = "Address type is verified/evidenced", example = "true")
+    private Boolean typeVerified;
+
+    @ApiModelProperty(name = "If address previously assessed then the latest assessment date and time", example = "2021-06-11T12:00:00")
+    private LocalDateTime latestAssessmentDate;
+
+    @ApiModelProperty(name = "Date and time that this address was created", example = "2021-06-11T13:00:00")
```
Comment (andymarke) [inline src/main/java/uk/gov/justice/digital/delius/data/api/Address.java:65]: Curious, why do you need these?

**Review by andymarke — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/justice/digital/delius/data/api/Address.java (hunk 1 above)
Comment (axle-h) [reply src/main/java/uk/gov/justice/digital/delius/data/api/Address.java:65]: Not sure why the offender manager needs them and there are some technical issues with the placement on the mock-up here but they're part of the design. On second thoughts though the created time is not required currently - just thought them best to come as a pair?!
image

Comment (andymarke) [reply src/main/java/uk/gov/justice/digital/delius/data/api/Address.java:65]: cool, not an issue, just curious - rare that clients need these dates so nice to see use cases when they do

**Pull request merged**